### DISCUSSION TITLE
Disable github config fetching on image start

### DISCUSSION THREAD

**Discussion by Steve-Brule** [haugene/docker-transmission-openvpn#2221] [Q&A] [answered]
Is there a way to completely disable this behavior? I use my own custom configs and the image often hangs on trying to fetch these configs that aren't needed for anything. Github works intermittently where my server is located.

I tried fiddling with GITHUB_CONFIG_SOURCE_REPO, but it seems that a call will have to be made to github no matter what this is set to.

**Comment by pkishino** [comment, marked as answer]
Set `VPN_CONFIG_SOURCE=custom`
